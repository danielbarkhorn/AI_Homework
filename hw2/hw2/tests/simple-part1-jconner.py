#!/usr/bin/python

import resolve

print(resolve.resolve(["or", "a", "b", "c"], ["not", "b"]))
print(resolve.resolve(["or", "a", "b", "c"], ["or", "b", ["not", "c"]]))
print(resolve.resolve(["or", ["not", "raining"], "wet ground"], "raining"))
print(resolve.resolve(["or", "a", "b"], "c"))
print(resolve.resolve(["or", ["not", "a"], ["not", "b"], ["not", "c"]],
              ["or", "a", "b", ["not", "d"]]))
print(resolve.resolve("a", ["not", "a"]))
