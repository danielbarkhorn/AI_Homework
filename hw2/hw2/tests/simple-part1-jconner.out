['or', 'a', 'c']
['or', 'a', 'b']
wet ground
False
False
[]
